### PR TITLE
Swipe to edit and delete friends. Friend description added in form & detail view.

### PR DESCRIPTION
**Pull request by davideast**
@mgonto 

Features added.

- Swipe to to get edit and delete options
- Add modal presents option for friend description
- Friend detail displays description

### CODE REVIEW

File: examples/firebase-sample/www/js/controllers.js [hunk 1]
```diff
@@ -35,20 +35,35 @@ angular.module('starter.controllers', [])
...
 
   $scope.showAddFriend = function() {
     $scope.modal.show();
-  }
+  };
 
   $scope.addFriend = function() {
-    Friends.add($scope.newFriend.name);
-    $scope.newFriendName = '';
+    debugger;
```
Comment (mgonto) [inline examples/firebase-sample/www/js/controllers.js]: Remove the debugger before merge please

**Comment by mgonto** [comment]
Awesome @davideast, thanks :).

Can you please remove the Debugger? I'll merge then :)

File: examples/firebase-sample/www/js/controllers.js (hunk 1 above)
Comment (davideast) [inline examples/firebase-sample/www/js/controllers.js]: I am useless without a linter.

Comment (mgonto) [inline examples/firebase-sample/www/js/controllers.js]: hahahaha no problem :P I can play as your linter (?)

**Pull request merged**